template/ci_verify.sh

```
@@ -29,33 +32,72 @@ uv pip install -r requirements.txt
+else
+    ALLOW_DIRTY="--allow-dirty"
+fi
+
+shopt -s nullglob
+for lock in ${ROOT}/services/*/runtime-lock.yaml; do
+    instance_dir=$(dirname "${lock}")
+    instance_name=$(basename "${instance_dir}")
+
+    # honour .ci_skip_checks
+    checks=${ROOT}/.ci_skip_checks
+    if [[ -f ${checks} ]] && grep -q "${instance_name}" ${checks}; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)